test/e2e/conformance_test.go

```diff
@@ -148,13 +148,13 @@ var _ = Describe("Conformance Tests", func() {
 			// The templates use WORKER_MACHINE_COUNT for linux machines for backwards compatibility so clear it
 			linuxWorkerMachineCount = 0
 
-			// Can only enable HostProcessContainers Feature gate in versions that know about it.
-			v122 := semver.MustParse("1.22.0")
+			// Can only enable HPAContainerMetrics Feature gate in versions that know about it.
+			v120 := semver.MustParse("1.20.0")
```

Comment:
More of a clarifying question for my understanding; 
-  Why are we not testing if Horizontal Pod Autoscaling Container Metrics (HPAContainerMetrics) is not set of `1.23` ? Or is this just a FeatureGate test to check if a feature gate called `HPAContainerMetrics` has been set since it will be removed starting `1.26` ?

Reply:
We need to set the `HPAContainerMetrics` feature gate to run some of the Windows tests and that feature gate was adde din v1.20.
I suppose we could remove the version check because I hope nobody is building clusters older than v1.20.
WDYT?

Reply:
+1 to removing the version check, especially in the test file